core/src/main/res/values/strings.xml

```diff
@@ -689,8 +689,8 @@
     <!-- Feed settings/information screen -->
     <string name="authentication_label">Authentication</string>
     <string name="authentication_descr">Change your username and password for this podcast and its episodes.</string>
-    <string name="feed_folders_label">Folders</string>
-    <string name="feed_folders_summary">Change the folders in which this podcast is displayed.</string>
+    <string name="feed_tags_label">Tags</string>
+    <string name="feed_tags_summary">Change the tags of this podcast.</string>
```

Comment:
When I'm reading this description, I have no mental model of it, at all. Folders are something that you experience frequently but tags are a lot more rare. A tag could be anything. When reading just this text, users cannot know what the setting will do. I would do something like "Categorize/group your subscriptions" but I'm not 100% happy about that text. @keunes do you have an idea how to best say this?

Reply:
How about 'tag group'? I feel that decreases some of the ambiguity of just tag. 
So the full text would be 'Change the tag groups of this podcast'?

Or we could just keep tag.

Reply:
Honestly I think tag is quite clear for basic user especially as it widely used for instance in Gmail. (And if needed it might be good to copy what Gmail use for label / text)

Still if really tag is not clear enough I agree with @peakvalleytech suggestion.  (But we weren't using folder group 😉)

Reply:
> as it widely used for instance in Gmail. 

My mail provider just has folders, no tags

> How about 'tag group'?

"Tag groups" is somehow duplicated. Maybe something like "Organize your list of subscriptions by adding tags"? It then doesn't simply repeat what the title says but instead explains what the setting is used for.

Reply:
This label is displayed on the settings page of a specific podcast, right?

What about "Change the tags of/on/for this podcast to help organize your subscriptions" ? (We should probably ask a native to select the right preposition).

Reply:
Tbh, i don't it really is an issue to use 'tags'. Let's just pick one and be consistent. I see we user tags with the wording:
Update podcast tags. 
We shouldn't underestimate the intelligence of the user. I'm sure they figure it out after a few uses. Besides tags is more for power users anyway. And if they're a power user, they probably know all about tags/folders/groups etc. No need to try to make it user friendly for monkeys.

Reply:
Ok, I checked with a colleague, and it should be 'of'
For me it's fine to just have "Change the tags of this podcast."

I guess the icon in the Podcast settings should be adapted accordingly. I would propose https://materialdesignicons.com/icon/tag-outline

Would this text & icon work for you, @ByteHamster?

Reply:
I like "Change the tags of this podcast to help organize your subscriptions" :+1: and the icon works for me

Reply:
![outline_label_black_24dp](https://user-images.githubusercontent.com/65185819/132783388-4eb07752-e542-4127-833b-c23548ae03ac.png)
![outline_label_off_black_24dp](https://user-images.githubusercontent.com/65185819/132783403-5a856669-0e65-4400-8166-cbe9dd38003a.png)

I initially wanted to used @keunes icon, but i have been working on the tag action for multi select. I have decided to separate the two actions. So there will be a separate multi select action for both 'add tag' and 'remove tag'. Thus, we need we can either create a new icon for removing tags or we can go with these two material icons. I think they look nicer anyways. I don't really like that dot on the other one. Adds clutter in my opinion. These two are much cleaner.

Reply:
Not sure I like those icons. Buy they do the job and if you've already used them in your other PR then let's go for this 🙂

Reply:
I found them not nice as well :-/
I would prefer something like these ones :
https://icons8.com/icon/119/price-tag
https://icons8.com/icon/12390/remove-tag

Reply:
@keunes @Matth7878. Alright, we can definitely go with these ones. No main problem was being able to find a matching icon for 'remove tag'. I was hoping we could stick to material icons. But if @ByteHamster is okay with them, i'll use them.

Reply:
Hmm. To be honest, I would prefer to use one from materialdesignicons.com because we already use other icons by them and already credit them

Reply:
Then no opposition from me since it will make it more coherent with other icons. 👍

Reply:
Ok. So we're back to the [tag-outline](https://materialdesignicons.com/icon/tag-outline) icon then :)

I'm not so much against the `label` icon that @peakvalleytech proposed, but there's no 'add' and 'remove' versions of it. For the `tag` there are
* https://materialdesignicons.com/icon/tag-minus-outline
* https://materialdesignicons.com/icon/tag-plus-outline